packages/swapper/src/swappers/thorchain/utils/makeSwapMemo/makeSwapMemo.ts

```diff
@@ -26,7 +26,12 @@ export const makeSwapMemo = ({
       details: { buyAssetId }
     })
 
-  const memo = `s:${thorId}:${destinationAddress}:${limit}:${THORCHAIN_AFFILIATE_NAME}:${THORCHAIN_AFFILIATE_BIPS}`
+  // bch hack
+  const parsedDestAddress = destinationAddress.includes('bitcoincash:')
+    ? destinationAddress.replace('bitcoincash:', '')
+    : destinationAddress
+
+  const memo = `s:${thorId}:${parsedDestAddress}:${limit}:${THORCHAIN_AFFILIATE_NAME}:${THORCHAIN_AFFILIATE_BIPS}`
```

Comment:
We have a `makeSwapMemo`, can't we use that?

Reply:
I dont follow???